app/views/contribute/AdventurerView.coffee

```diff
@@ -5,4 +5,6 @@ template = require 'templates/contribute/adventurer'
 module.exports = class AdventurerView extends ContributeClassView
   id: 'adventurer-view'
   template: template
-  contributorClassName: 'adventurer'
+
+  initialize: ->
+    @contributorClassName = 'adventurer'
```

Comment:
Technically this change is unnecessary, since the property is a string, not an array or object. But either way works!